docs/manual/developer_guide.adoc

```diff
@@ -815,6 +815,7 @@ The <tt>/var/tmp</tt> directory is a world-writable directory used for temporary
 +
 When deciding on severity levels, it is best to follow the following guidelines:
 .Table Vulnerability Severity Category Code Definitions
++
```

Comment:
Display rich format option from Github shows a lot of changes, but this `+` was missing and now https://github.com/ComplianceAsCode/content/pull/5144/files#diff-bcd4620adacf98f7425fd2f913933222R834 shows correctly under the table.